### ISSUE TITLE
Modal opens at bottom of the screen.

### ISSUE THREAD

**Issue by gaurovsoni** [saadeghi/daisyui#138]
Details: when using with windicss, the modal opens up at the bottom of the screen. 
![Screenshot from 2021-08-18 17-17-28](https://user-images.githubusercontent.com/9216097/129893192-ea3992f3-3802-4d49-afa6-0296bfeaeffe.png)

**Comment by gringofe** [comment]
Seems like this has been fixed in windicss!

**Comment by saadeghi** [comment]
@gaurovsoni Yes. I just tested the latest version of Windi CSS and the modal works perfectly now 🥳
